primitives/ethereum/src/serde_utils.rs

```diff
@@ -0,0 +1,80 @@
+pub mod serde_fork_version {
+    use serde::Deserialize;
+
+    use crate::beacon::ForkVersion;
+
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<ForkVersion, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        let value = String::deserialize(deserializer)?;
+        let value = if value.len() > 1 && value[0..2] == *"0x" {
```

Comment:
what if `value.len() == 0` ?
In this case code goes to `else` section and we continue to work with empty address. Is it ok?

Reply:
And should we validate the address length?